controllers/authorsController.js

```diff
@@ -0,0 +1,61 @@
+const client = require('../config/connection');
+const { internalServerError, getDataByField } = require('../utils/utils');
+
+exports.getAuthorsByCentury = async (req, res) => {
+  try {
+    const { century } = req.params;
+
+    const { rows } = await client.query(
+      'SELECT * FROM author WHERE birth_year >= $1 AND birth_year <= $2',
+      [century * 100, century * 100 + 100]
```

Comment:
this century calculation isn't right, take an example as 18, your statement checks 1800<=century<=1900, which is already the 19th century, but 1800 is in 18th century, so check both the calculation and conditions